package.json

```diff
@@ -6,8 +6,11 @@
     "grunt-contrib-qunit": "^4.0.0",
     "phantomjs-prebuilt": "2.1.15"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "sass": "^1.29.0"
+  },
   "scripts": {
-    "test": "grunt test"
+    "test": "grunt test",
```

Comment:
Is there anyway we can add a github action step that ensures that scss was compiled into css for all PRs? That way we don't have to accidentally worry about people forgetting to run this before they open a PR.

@ssbarnea what do you think?

Reply:
but we might need reproducible builds, then. which means npm's `package-lock.json`  should be under version control.

Reply:
Let's not bloat this PR. Can you create a follow up issue for this @jkowalleck ?

Reply:
added #392 to keep track of this topic.